transforms/dialog-full-screen-app-wrapper/__testfixtures__/WithChildrenAsAnExpressionContainerWithNoAppWrapper.output.js

```diff
@@ -0,0 +1,63 @@
+import DialogFullScreen from "carbon-react/lib/components/dialog-full-screen";
+import AppWrapper from "carbon-react/lib/components/app-wrapper";
+import FooWrapper from "carbon-react/lib/components/foo-wrapper";
+
+export default (props) => {
+  const content = (props) => (
+    <FooWrapper> 
+      { props.children }
+    </FooWrapper> 
+  );
+
+  return (
+    <DialogFullScreen><AppWrapper>
```

Comment:
It would be a nit nicer to have this `AppWrapper` added on the next line - did you look into doing that?

Reply:
Yeah it's possible to add `\n` but I decided not to as the linter/ prettier will sort it out when they run. The reason I decided not to was that it would also add newlines even when the component is rendered as a single line statement

eg: `<DialogFullScreen>{ content }</DialogFullScreen>`
->
`<DialogFullScreen>
<AppWrapper>{ content }
</AppWrapper>
</DialogFullScreen>`

Reply:
I think Ed is right here, linters and prettiers will format this correctly no need to ad extra cases into it.

Reply:
What if they don't have formatters on their project? We don't have them yet in Carbon.

I had to fix an issue with my recent codemod that would have been removed with formatters for this exact reason.

I've commented below as well, you should be able to check whether the `DialogFullScreen` opening element is on a line by itself, and if so you can add the `\n`, if not just add it on the same line